env/Pa_interpolate_simple.yml

```diff
@@ -0,0 +1,49 @@
+name: Pa_interpolate_simple
+channels:
+  - anaconda
+  - conda-forge
+  - defaults
+dependencies:
+  - conda-forge::python=3.5.5
+  - conda-forge::keras=2.1.6
+  - conda-forge::tensorboard=1.10.0
+  - conda-forge::tensorflow=1.10.0
+  - conda-forge::pandas=0.22.0
+  - conda-forge::numpy=1.15.2
+  - conda-forge::numexpr=2.6.4
+  - conda-forge::scikit-learn=0.19.1
+  - conda-forge::scipy=1.1.0
+  - conda-forge::seaborn=0.8.1
+  - conda-forge::hdf5=1.10.1
+  - conda-forge::ggplot=0.11.5
+  - conda-forge::graphviz=2.38.0
+  - conda-forge::python-graphviz=0.8.4
+  - conda-forge::umap-learn=0.3.2
+  - conda-forge::six=1.11.0
+  - conda-forge::jupyter=1.0.0
+  - conda-forge::ipykernel=4.8.1
+  - conda-forge::ipywidgets=6.0.0
+  - conda-forge::widgetsnbextension=2.0.0
+  - conda-forge::nb_conda_kernels=2.1.0
+  - conda-forge::nbconvert=5.5.0
+  - anaconda::pydot=1.2.3
+  - anaconda::gxx_linux-64
+  - anaconda::libstdcxx-ng=8.2.0
+  - conda-forge::libgcc=7.2.0
+  - anaconda::libgcc-ng=8.2.0
+  - conda-forge::r-base=3.4.1
+  - conda-forge::r-irdisplay=0.6.2
+  - conda-forge::r-irkernel=0.8.11
+  - conda-forge::r-ggplot2=2.2.1
+  - conda-forge::r-dplyr=0.7.6
+  - conda-forge::r-tidyr=0.8.0
+  - conda-forge::r-readr=1.1.1
+  - conda-forge::r-readxl=1.1.0
+  - conda-forge::r-cowplot=0.9.3
+  - conda-forge::r-heatmap3=1.1.1
+  - conda-forge::r-ggrepel=0.8.0
+  - conda-forge::r-optparse=1.6.0
+  - pip:
+    - matplotlib==3.0.0
```

Comment:
Would move matplotlib and tqdm into the dependencies section with the conda-forge channel.

Reply:
So I'm not as proficient in conda environments, but I originally did have matplotlib in the dependency section and I got a failure to setup the environment.  I moved it based on a blog post.  But very happy to sit down and learn more about how to better setup these environment files in the future